protos/flyteidl/external/external_event.proto

```diff
@@ -0,0 +1,75 @@
+syntax = "proto3";
+
+package flyteidl.external;
+
+option go_package = "github.com/lyft/flyteidl/gen/pb-go/flyteidl/external";
+
+import "flyteidl/core/execution.proto";
+import "flyteidl/core/identifier.proto";
+import "flyteidl/core/catalog.proto";
+import "flyteidl/core/workflow_closure.proto";
+import "flyteidl/core/compiler.proto";
+import "flyteidl/admin/event.proto";
+import "flyteidl/admin/common.proto";
+import "flyteidl/admin/execution.proto";
+import "flyteidl/admin/node_execution.proto";
+import "flyteidl/admin/task_execution.proto";
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/struct.proto";
+
+message ExecutionEventClosure {
+    flyteidl.core.ConnectionSet connections = 1;
+
+    flyteidl.core.LiteralMap computed_input = 2;
+
+    core.Identifier launch_plan        = 3;
+    flyteidl.admin.Annotations annotations = 4;
+    // parent_node_execution
+    flyteidl.admin.ExecutionMetadata metadata = 5;
+    flyteidl.admin.Labels label = 6;
+}
+
+message WorkflowEvent {
+   // Workflow execution id
+   flyteidl.admin.Execution execution = 1;
+
+   ExecutionEventClosure workflow_event_closure = 2;
+}
+
+message NodeEventClosure {
+    string spec_node_id = 1;
+    // Only available when the node is a task node and node is finished
+    oneof taskExecution {
+        flyteidl.admin.TaskExecution finalTaskExecution = 2;
```

Comment:
is this not redundant with https://github.com/flyteorg/flyteidl/blob/9287252f43909407a014990b82d7f317dbe5562c/protos/flyteidl/admin/node_execution.proto#L138 that is encapsulated in spec_node below?